src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -75,116 +73,174 @@ public bool TryGetTemperature(out Temperature temperature)
         /// <remarks>
         /// If last read was not successfull, it returns double.MaxValue
         /// </remarks>
-        public double Humidity => (_dhtType == DhtType.Dht11) ? GetHumidityDht11() : GetHumidityDht22();
+        public double Humidity
```

Comment:
starts to feel like we should have a base class (no strong opinion yet but still looking at the PR)

Reply:
ok after finishing I think it's fine as is - I was expecting more switches later